libs/application/templates/children-residence-change/src/fields/Overview/index.tsx

```diff
@@ -24,24 +28,36 @@ const Overview = ({ application }: FieldBaseProps) => {
 
   const [
     createPdfPresignedUrl,
-    { loading: loadingUrl, data: response },
+    { loading: createLoadingUrl, data: createResponse },
   ] = useMutation(CREATE_PDF_PRESIGNED_URL, {
     onError: (e) => console.log('error', e),
   })
 
+  const [
+    getPresignedUrl,
+    { data: getResponse, loading: getLoadingUrl },
+  ] = useLazyQuery(GET_PRESIGNED_URL)
+
   useEffect(() => {
-    createPdfPresignedUrl({
+    let input = {
       variables: {
         input: {
           id: application.id,
           type: PdfTypes.CHILDREN_RESIDENCE_CHANGE,
         },
       },
-    })
-  }, [application.id, createPdfPresignedUrl])
+    }
+
+    application.state === ApplicationStates.DRAFT
+      ? createPdfPresignedUrl(input)
+      : getPresignedUrl(input)
+  }, [application.id, createPdfPresignedUrl, getPresignedUrl])
 
   const pdfUrl =
-    response?.createPdfPresignedUrl?.attachments?.[
+    createResponse?.createPdfPresignedUrl?.attachments?.[
+      PdfTypes.CHILDREN_RESIDENCE_CHANGE
+    ] ||
+    getResponse?.getPresignedUrl?.attachments?.[
```

Comment:
Not loving this, but discussed with Villi and we will do a refactor once frontend for signature is done since it is touching on similar things.